spec/order_spec.rb

```diff
@@ -0,0 +1,30 @@
+require 'order'
+
+describe Order do
+  subject(:order) { described_class.new }
+  let(:my_order) { { 'Chicken Shish' => 12, } }
+  #it { expect(described_class).to be < Menu }
+
+before do 
+  order.add_dish_to_order('Chicken Shish', 2)
+end
+
+
+  it 'is expected to select several dishes from the menu' do
+    #my_order.foods_ordered('chips', 2)
+    expect(order.my_order).to eq my_order
+  end
+
+  it 'allows customer to add new dishes to the cart' do
+    order.add_dish_to_order('Chicken Shish', 2)
```

Comment:
Is this necessary? Doesn't the before take care of adding something to the order? I'll admit I'm still slightly uncertain about the usage of before, so its more of a question than anything.